pkg/reconciler/buildrun/resources/taskrun.go

```diff
@@ -113,7 +126,7 @@ func GenerateTaskSpec(
 		},
 	}
 
-	generatedTaskSpec.Results = getTaskSpecResults()
+	generatedTaskSpec.Results = append(generatedTaskSpec.Results, getTaskSpecResults()...)
```

Comment:
I think other common results such as the digest and size of the output image are also provided in the results.go file. I therefore prefer that you move the two results defined here to that file, and we keep taskrun.go short.

Reply:
ok done